src/map/utils/petutils.cpp

```diff
@@ -1096,8 +1096,10 @@ namespace petutils
 
         Pet_t* PPetData = g_PPetList.at(PetID);
 
-        if (PMaster->objtype == TYPE_PC)
+        if (PMaster->objtype == TYPE_PC){
```

Comment:
logout should pretty much be the same as zoning out so surprised this isn't already a thing honestly.